watertap/unit_models/zero_order/nanofiltration_zo.py

```diff
@@ -72,3 +72,11 @@ def load_parameters_from_database(self, use_default_removal=False):
         # TODO: Do not have parameters for these yet - need to add to datafile
         self.deltaP_treated.fix(0)
         self.deltaP_byproduct.fix(0)
+
+    def _get_performance_contents(self, time_point=0):
+        perf_dict = super()._get_performance_contents(time_point=0)
+
+        perf_dict["vars"]["Electricity Demand"] = self.electricity[time_point]
+        perf_dict["vars"]["Electricity Intensity"] = self.electricity_intensity
```

Comment:
There might be better alternatives for this, but should we consider adding units to the reported variable names for clarity? Perhaps there should eventually be another column for units in the report table. Just something that I wanted to point out for a future PR, and I think (not 100% sure) I saw an issue or PR somewhere on IDAES that mentions adding units to reports.